pkg/harvester/components/HarvesterUpgradeHeader.vue

```diff
@@ -2,7 +2,7 @@
 import { NODE } from '@shell/config/types';
 import { HCI } from '../types';
 import { allHash } from '@shell/utils/promise';
-import { HCI as HCI_ANNOTATIONS } from '@/pkg/harvester/config/labels-annotations';
+import { HCI as HCI_ANNOTATIONS } from '~/pkg/harvester/config/labels-annotations';
```

Comment:
Preferably all these changes from these should stay as `@/pkg`, I think that might break things somewhere. The `check-plugin` github workflow will fail if there's any in i think shell or a test pkg.

Reply:
As a fall back, they could be made relative rather than absolute

Reply:
I think this is related to the configuration we have in this project which is messing up also with the intellisense. Every time there's an automatic import, it changes to it. Should we create a further issue or is sufficient the [existing one](https://github.com/rancher/dashboard/issues/7357) (maybe with higher weight)?

Reply:
I was able to fix this at least at build time with a new alias. It doesn't look like it broke anything.